packages/AutoTDD-Core.package/ATDDTestingEntity.class/class/colorDictionary.st

```diff
@@ -1,10 +1,11 @@
 constants
 colorDictionary
-	
+	| currentTheme |
+	currentTheme := UserInterfaceTheme current.
 	^ IdentityDictionary
 		newFrom: {
-			#error -> Color red.
-      		#failure -> Color orange.
-			#successful -> Color green.
-			#pending -> Color gray
+			#error -> (currentTheme properties at: TestRunner->#errorColor).
+      		#failure ->  (currentTheme properties at: TestRunner->#failureColor).
+			#successful -> (currentTheme properties at: TestRunner->#passColor).
+			#pending -> Color gray.
```

Comment:
nit: dot not necessary

```suggestion
			#pending -> Color gray
```

Also does `Color gray` work fine for all themes? Is there an alternative in the `TestRunner Colors`?

Reply:
No specific TestRunner neutral color, however will change to match all default text color for consistency sake (even though the gray color did work quite well on all themes).